bundler/lib/bundler/source/path.rb

```diff
@@ -225,7 +225,7 @@ def generate_bin(spec, options = {})
         # Some gem authors put absolute paths in their gemspec
         # and we have to save them from themselves
         spec.files = spec.files.map do |p|
-          next p unless p =~ /\A#{Pathname::SEPARATOR_PAT}/
```

Comment:
:thinking: When touching this line, maybe it is good time to rename block var `p` to not shadow `Kernel#p`. Maybe to `path`?

Reply:
Sure, why not.